### DISCUSSION TITLE
stop exporting go level metrics

### DISCUSSION THREAD

**Discussion by Khushbukela** [prometheus/client_golang#956] [Q&A] [answered]
How to not export go-level metrics?

**Comment by beorn7** [comment, marked as answer]
You have to use your own registry rather than the default one.

Create it with https://pkg.go.dev/github.com/prometheus/client_golang@v1.11.0/prometheus#NewRegistry and then only register the collectors you really want (i.e. do not register https://pkg.go.dev/github.com/prometheus/client_golang@v1.11.0/prometheus/collectors#NewGoCollector ).
